modules/nodes-custom-scheduler-packaging.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-custom-scheduler.adoc
+
+[id="nodes-custom-scheduler-packaging_{context}"]
+= Packaging and deploying a custom scheduler
+
+To include a custom scheduler in your cluster, create an image and include that image in a deployment.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a scheduler binary.
++
+[NOTE]
+====
+Information on how to create the scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation.
+====
+
+.Procedure
```

Comment:
Wait its out of scope, here let me show you how to build an image?  Can we re-expore if this procedure is needed?

Reply:
I've removed the first steps and have added to build the image/push to a registry as a prereq